packages/peregrine/lib/talons/SearchPage/useSearchPage.js

```diff
@@ -0,0 +1,68 @@
+import { useCallback, useEffect } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useAppContext } from '@magento/peregrine/lib/context/app';
+import { useHistory, useLocation } from 'react-router-dom';
+
+import getQueryParameterValue from '../../util/getQueryParameterValue';
```

Comment:
Since we're moving these utilities, I think this is a good opportunity to rename them. I think the we should avoid the term "query parameter":

- the modern name for them appears to be [`SearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)
- `query` is extremely overloaded thanks to GraphQL terminology

Would love to see this renamed to `getSearchParamValue`, and similar renames for other utils and local variables.

Reply:
Discovered that you created this utility in Peregrine already, so I went ahead and just deleted this one 💣